etrago/cluster/networkclustering.py

```diff
@@ -910,6 +922,8 @@ def normed(x):
         .set_index("T" + trafo_index),
         "Line",
     )
+    network.lines.lifetime = 40
```

Comment:
Is this line still needed?

Reply:
Fixed in adf6dff52902271495153c290ed2c01ba9ed0ce6